src/app/header/header.component.scss

```diff
@@ -119,7 +119,7 @@
       z-index: 1001; // +1 to keep above dropdown menus
       padding: 0.875rem 1rem;
       border-top: 1px solid $mn-border-color;
-      color: #fcba18;
+      color: $header-font-color;
```

Comment:
Just another little nudge to make these colour variables more comprehendible, possibly by including the colour name in the variable name.